src/index.html

```diff
@@ -2,16 +2,173 @@
+              >
+            </div>
+            <div class="input">
+              <label for="pwd">Password:</label>
+              <input 
+              type="password" 
+              id="pwd" 
+              name="pwd" 
+              autocomplete="off" 
+              minlength="12" 
+              maxlength="100"
+              >
+            </div>
+          </form>
+        </fieldset>
+      </div>
+
+      <div class="fieldset">
+        <fieldset>
+          <legend>An interesting fact about you!</legend>
+          <form>
+            <div class="input">
+              <label for="cats">Do you love cats?</label>
+              <label>
+                <input type="radio" id="yes" name="cats">
+                Yes
+              </label>
+              <label>
+                <input type="radio" id="no" name="cats">
+                No
+              </label>
+            </div>
+            <div class="input">
+              <label for="favcolor">What is your favorite color?</label>
+              <input 
+              type="color" 
+              id="favcolor" 
+              name="favcolor" 
+              value="black"
+              >
+            </div>
+            <div class="input">
+              <label for="appt">What time do you go to bed?</label>
+              <input 
+              type="time" 
+              id="appt" 
+              name="appt" 
+              step="1"
+              >
+            </div>
+            <div class="input">
+              <label for="cars">What are your favorite brand of cars?</label>
+              <select id="cars" name="cars" multiple>
+                <option value="volvo">BMW</option>
+                <option value="saab">Audi</option>
+                <option value="fiat">Lada</option>
+                <option value="0"> </option><br>
+              </select> 
+            </div>
+            <div class="input">
+              <label for="rate">How do you rate our work?</label>
+              <input 
+              type="range"
+              id="rate" 
+              min="12" 
+              max="100" 
+              value="0"
+              >
+            </div>
+          </form>
+        </fieldset>
+      </div>
+
+      <div class="fieldset">
+        <fieldset>
+          <legend>Additional info:</legend>
+          <form>
+            <div class="input">
+              <label for="comments">Comments:</label>
+              <textarea id="comments" name="comments" autocomplete="off">
+              </textarea>
+            </div>
+            <div class="input">
+              <label for="recommend">Would you recommend us?</label>
+              <select id="recommends" name="cars">
+                <option value="yes">yes</option>
+                <option value="no">no</option>
+              </select>
+            </div>
+          </form>
+        </fieldset>
+      </div>
+
+
+
+      <button type="submit">
+        Submit
+      </button>
+
+
+
+    </form>
```

Comment:
```suggestion
      </div>

      <button type="submit">
        Submit
      </button>

    </form>
```
Не потрібно робити так багато вільних рядків